soc/arm/atmel_sam0/common/soc_samd2x.c

```diff
@@ -55,8 +55,13 @@ static void xosc32k_init(void)
 
 static void osc32k_init(void)
 {
+#ifdef FUSES_OSC32K_CAL_ADDR
 	u32_t fuse = *(u32_t *)FUSES_OSC32K_CAL_ADDR;
 	u32_t calib = (fuse & FUSES_OSC32K_CAL_Msk) >> FUSES_OSC32K_CAL_Pos;
+#else
+	u32_t fuse = *(u32_t *)FUSES_OSC32KCAL_ADDR;
```

Comment:
This looks like a bug in the header files. Maybe we should report it to Microchip?